my-app/src/DataStore.jsx

```diff
@@ -23,7 +26,9 @@ class DataStore {
      // Collection names changable so you can do some test dicking around
      this._personCollection = "persons";
      this._tagCollection = "tags";
-     this.firestore = firebase.firestore();
+
+     this.firestore = ENABLE_LOGIN ? null : firebase.firestore();
```

Comment:
Right now `ENABLE_LOGIN` is null everywhere right?

Reply:
enable login is false everywhere as dictated by the line on the bottom of the file